src/components/Navbar/Navbar.vue

```diff
@@ -60,8 +66,10 @@ const toggleDrawer = () => {
   <v-app-bar class="ios-padding">
     <v-app-bar-nav-icon @click="toggleDrawer" />
     <v-app-bar-title>
-      <span v-if="$vuetify.display.smAndUp" class="text-accent">Vue</span>
-      <span v-if="$vuetify.display.smAndUp">Torrent</span>
+      <v-app-bar-title style="display: inline-flex; width: min-content; cursor: pointer" @click="goHome">
```

Comment:
Can you explain the inline styles?

Reply:
Display: inline-flex allows the active filters div to sit side by side with the app-bar title since its now technically two separate divs.

Width: min-content resizes the inner v-app-bar-title to be the same size as the title representing the area you can click to return to the homepage. The reason why I created a wrapper element for the title is because v-app-bar-title spans the width of the entire app-bar and cannot be changed without also changing the top-actions' positioning.

Cursor: pointer make the cursor a pointer so that users know that you can click on the title to return to the homepage.

If you'd prefer this within the styles.scss file I can make that change.

Reply:
Apart from that it's great!

Reply:
No need to declare that class globally, declare it inside the component as it will only be used there.

Reply:
Ah that makes more sense, should be fixed now.